api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -195,6 +195,10 @@ model ApiProfile {
   /** The internet visibility of the OpenShift API server */
   @visibility(Lifecycle.Read, Lifecycle.Create)
   visibility?: Visibility = "public";
+
+  /** The IP ranges authorized to access the Kubernetes API server. */
+  @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
+  authorizedIpRanges?: string[];
```

Comment:
Do we know if Hypershift supports this in some way?

Reply:
It does not atm. This is how we originally had capture this functional req https://issues.redhat.com/browse/CNTRLPLANE-203

Are we saying this would be a list of CIDRs? Is this mutable? Is there an upper limit?
Let's flesh out all those details and capture in CNTRLPLANE that supersedes 203